.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -1,46 +1,53 @@
-name: Levitate
+name: Levitate / Report breaking changes
 
-on: push
+on:
+  workflow_run:
+    workflows: ["Levitate / Detect breaking changes"]
+    types: [completed]
 
 jobs:
-  build:
-    name: Detecting breaking changes
+  notify:
+    name: Report
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
```

Comment:
**Question:** do we need to check out the code here?

As this only runs code from `main` and there are no NPM life-cycle scripts triggered I think it shouldn't cause any harm, just asking to make sure we are extra safe :)